Clients/GUI/Shared/Header.razor

```diff
@@ -56,18 +56,18 @@
     </nav>
 </div>
 
+
 @code {
-    private string defaultImgSrc = "Picture/Avatar.svg";
+	private string defaultImgSrc = "Picture/Avatar.svg";
```

Comment:
hm, does not seem aligned

Reply:
strange, i don't see this in visual studio